stacks-signer/src/v0/signer.rs

```diff
@@ -538,15 +538,7 @@ impl Signer {
             .signer_db
             .block_lookup(self.reward_cycle, &signer_signature_hash)
         {
-            Ok(Some(block_info)) => {
```

Comment:
I think removing this will cause lots of errors about failing to mark locally accepted/locally accepted. I would just short circuit here and send the response. Similarly with below.